frontend/src/components/visual-editor/form/inputs/triggers/PatternInput.tsx

```diff
@@ -268,19 +268,14 @@ const PatternInput: FC<PatternInputProps> = ({
             {...registerInput(t("message.regex_is_invalid"), idx, {
               validate: (pattern) => {
                 try {
-                  if (
-                    pattern.at(0) === "/" &&
-                    pattern.at(-1) === "/" &&
-                    typeof pattern === "string"
-                  )
-                    new RegExp(pattern.slice(1, -1));
+                  if (isRegex(pattern)) new RegExp(pattern.slice(1, -1));
```

Comment:
If it doesn't starts/ends with "/", this will bypass the check. 

I think it should be 
```suggestion
                  new RegExp(isRegex(pattern) ? pattern.slice(1, -1) : pattern);
```

Reply:
Hi @marrouchi, i think that we use react-hook-form setValueAs register option that will executed before the validation process to be sure that the pattern is always starting and ending with '/' if i'm not mistaking.
![image](https://github.com/user-attachments/assets/9d72f519-2449-4f4b-826c-7a5db7baa319)
 Reference:
 https://react-hook-form.com/docs/useform/registe